src/Geosuggest.jsx

```diff
@@ -423,8 +423,9 @@ class Geosuggest extends React.Component {
       input = <Input className={this.props.inputClassName}
         ref={i => this.input = i}
         value={this.state.userInput}
-        ignoreEnter={!this.state.isSuggestsHidden}
+        submitOnEnter={!this.state.isSuggestsHidden}
```

Comment:
Better rename to `doNotSubmitOnEnter`. Changing name and meaning of the var leads to confusion.

Reply:
Done